app/controllers/api/v1/accounts_controller.rb

```diff
@@ -5,51 +5,39 @@ class Api::V1::AccountsController < ApiController
   before_action -> { doorkeeper_authorize! :follow }, only: [:follow, :unfollow, :block, :unblock, :mute, :unmute]
   before_action :require_user!, except: [:show]
   before_action :set_account
+  before_action :set_relationship_map, except: [:show]
 
   respond_to :json
 
   def show; end
 
   def follow
     FollowService.new.call(current_user.account, @account.acct)
-    set_relationship
     render :relationship
   end
 
   def block
     BlockService.new.call(current_user.account, @account)
-
-    @following       = { @account.id => false }
```

Comment:
These explicit values are a cheat. Because we know exactly what the relationship we're gonna get after the above service call, we can just hardcode the value instead of doing a roundtrip to the database.